README.md

```diff
@@ -17,9 +17,9 @@ and efficient manner.
 
 The following languages are currently supported:
 
-* Java
-* JavaScript, TypeScript
-* .NET
+* Java ![](docs/src/_images/java.png)
+* JavaScript ![](docs/src/_images/js.png), TypeScript ![](docs/src/_images/ts.png)
+* .NET ![](docs/src/_images/dotnet.png)
```

Comment:
Can we find logos with better resolution. This looks pretty crappy

Reply:
It's probably due to scaling as I wanted them all to be no larger than 24 x 24 pixels. That may be too small for the ones with text, such as .NET.

Reply:
Okay, I bumped them to 40x40 pixels:

![image](https://user-images.githubusercontent.com/16107690/47792491-62e52b00-dcd9-11e8-99c0-041cc3e9c98d.png)

Reply:
Sorry this looks horrible. Maybe we can just put the logos side by side before/after the bulletted list...